cdxev/__main__.py

```diff
@@ -712,6 +714,66 @@ def create_build_public_bom_parser(
     return parser
 
 
+# noinspection PyUnresolvedReferences,PyProtectedMember
+def create_init_sbom_parser(
+    subparsers: argparse._SubParsersAction,
+) -> argparse.ArgumentParser:
+    parser = subparsers.add_parser(
+        "init-sbom",
+        help=("Provides the first draft of an SBOM for manual completion."),
+        description=(
+            "Creates an initial SBOM draft, CycloneDX version1.6, "
+            "to work with, containing the most basic fields. "
+            "Optionally for some fields values can be submitted to the command. "
+            "Those fields are: "
+            "The name for one author of the SBOM (metadata.authors[0].name) "
+            "The name of the supplier of the software (metadata.component.supplier.name) "
+            "The name of the software (metadata.component.name) "
+            "The version of the software (metadata.component.version)."
+        ),
+        usage=(
+            "cdx-ev init-sbom [-h] [--name <name of the component>] "
+            "[--authors <name of the author>] "
+            "[--supplier <supplier of the component>] "
+            "[--version <version of the component>] "
+            "[--output <file>]"
+        ),
+    )
+    submitted_values = parser.add_argument_group(
+        "target",
+        description=(
+            "Submitted values that will be written into the SBOM draft. "
+            "Field values like the name and version of the software (--name and --version), "
+            "the supplier of the software (--supplier-software) "
+            "or the supplier of the SBOM (--supplier-sbom) "
+            "can be submitted to the program and will be written into the provided draft."
+        ),
+    )
+    submitted_values.add_argument(
+        "--name",
+        metavar="<name>",
+        help=("The name of the component."),
+    )
+    submitted_values.add_argument(
+        "--version",
+        metavar="<version>",
+        help=("The component version."),
+    )
+    submitted_values.add_argument(
+        "--supplier",
+        metavar="<supplier-software>",
+        help=("The name of the organization that supplied the component."),
+    )
+    submitted_values.add_argument(
+        "--authors",
+        metavar="<supplier-sbom>",
+        help=("The person who created the BOM."),
```

Comment:
would use "SBOM", I know in the cdx-specification it's BOM but only because cdx supports also e.g. HBOMs + you use "SBOM" the whole time and the `metavar` even included "SBOM".

Reply:
Done